### DISCUSSION TITLE
Option to disable caching for fetch() requests on model assets

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#282] [Q&A]
The browser (or Chromium Electron process) is caching assets by default when it can. Give the user the option to bypass caching.

**Comment by jtbandes** [comment]
Why is this desirable?

**Comment by jhurliman** [comment]
A user reported there is no other way of fetching updated assets in their particular environment. Refreshing the Studio webpage still caused the old versions of the assets to be fetched and rendered. It's possible their server is misconfigured (sending improper ETag headers?) but the request was to have control over Studio's caching and/or fetching behavior for 3D panel assets.
